[2025-03-09T03:29:43.228+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: 4Entrenamiento_model.validar_modelo scheduled__2025-03-09T03:10:00+00:00 [queued]>
[2025-03-09T03:29:43.236+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: 4Entrenamiento_model.validar_modelo scheduled__2025-03-09T03:10:00+00:00 [queued]>
[2025-03-09T03:29:43.236+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 2
[2025-03-09T03:29:43.247+0000] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): validar_modelo> on 2025-03-09 03:10:00+00:00
[2025-03-09T03:29:43.251+0000] {standard_task_runner.py:57} INFO - Started process 506 to run task
[2025-03-09T03:29:43.254+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', '4Entrenamiento_model', 'validar_modelo', 'scheduled__2025-03-09T03:10:00+00:00', '--job-id', '23', '--raw', '--subdir', 'DAGS_FOLDER/Entrenamiento_mode.py', '--cfg-path', '/tmp/tmpq8dx7pwn']
[2025-03-09T03:29:43.256+0000] {standard_task_runner.py:85} INFO - Job 23: Subtask validar_modelo
[2025-03-09T03:29:43.310+0000] {task_command.py:410} INFO - Running <TaskInstance: 4Entrenamiento_model.validar_modelo scheduled__2025-03-09T03:10:00+00:00 [running]> on host d79f96be65f0
[2025-03-09T03:29:43.374+0000] {taskinstance.py:1570} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='4Entrenamiento_model' AIRFLOW_CTX_TASK_ID='validar_modelo' AIRFLOW_CTX_EXECUTION_DATE='2025-03-09T03:10:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-09T03:10:00+00:00'
[2025-03-09T03:29:43.420+0000] {python.py:183} INFO - Done. Returned value was: {'mejor_modelo': 'KNN', 'train_accuracy': 1.0, 'test_accuracy': 1.0, 'todos_resultados': {'KNN': {'train_accuracy': 1.0, 'test_accuracy': 1.0}, 'LogReg': {'train_accuracy': 1.0, 'test_accuracy': 1.0}, 'LogRegCV': {'train_accuracy': 1.0, 'test_accuracy': 1.0}}}
[2025-03-09T03:29:43.438+0000] {taskinstance.py:1373} INFO - Marking task as SUCCESS. dag_id=4Entrenamiento_model, task_id=validar_modelo, execution_date=20250309T031000, start_date=20250309T032943, end_date=20250309T032943
[2025-03-09T03:29:43.465+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2025-03-09T03:29:43.482+0000] {taskinstance.py:2674} INFO - 0 downstream tasks scheduled from follow-on schedule check
